examples/integrations/server.js

```diff
@@ -0,0 +1,141 @@
+const express = require("express");
+const { makeExecutableSchema } = require("graphql-tools");
+const queries = require("./queries");
+const axios = require("axios");
+const {
+  ApolloVoyagerServer,
+  gql
+} = require("../../packages/apollo-voyager-server");
+const { GraphQLScalarType } = require("graphql");
+const { Kind } = require("graphql/language");
+
+// Types
+const typeDefs = gql`
+  scalar IntOrUnknown
+
+  type CharacterInfo {
+    name: String
+    surname: String
+    version: Int
+    height: IntOrUnknown
+    mass: IntOrUnknown
+    hair_color: String
+    skin_color: String
+    eye_color: String
+    homeworld: String
+    gender: String
+    genderFromName: GenderInfo
+  }
+
+  type GenderInfo {
+    firstName: String
+    lastName: String
+    scale: Float
+    gender: String
+  }
+
+  type Query {
+    getCharacterInfo(id: ID!): CharacterInfo
+    getGenderFromName(name: String, surname: String): GenderInfo
+  }
+`;
+
+const getGenderFromName = async (obj, args, context, info) => {
+  try {
+    const { name, surname } = args;
+    const response = await axios.get(
+      `http://api.namsor.com/onomastics/api/json/gender/${name}/${surname}`,
+      {
+        headers: {
+          "Content-Type": "application/json;charset=UTF-8"
+        }
+      }
+    );
+
+    const { data } = response;
+    return data;
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+const getCharacterInfo = async (obj, args, context, info) => {
+  try {
+    const response = await axios.get(`http://swapi.co/api/people/${args.id}`, {
+      responseType: "json",
+      headers: {
+        "Content-Type": "application/json;charset=UTF-8"
+      }
+    });
+    const { data } = response;
+    const [name, surname] = data.name.split(" ");
+    return { ...data, name, surname };
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+// Resolver functions. This is our business logic
+const resolvers = {
+  Query: {
+    getGenderFromName,
+    getCharacterInfo
+  },
+  CharacterInfo: {
+    genderFromName: async (obj, args, context, info) =>
+      await getGenderFromName(
```

Comment:
Not sure if it's the right approach to do the "sub-queries" in the resolvers. Some GraphQL expert might help...

Reply:
I would say remove this as purpose of this example is not related with sub-queries. It should be clearly show what it needs to show.